sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md

```diff
@@ -4,12 +4,19 @@
 
 ### Features Added
 
+- Introduced modern versions of the existing LRO methods in the `TextAnalyticsClient` that use the modern LRO pattern:
```

Comment:
I wouldn't call them "modern versions of the existing LRO methods". That's exposes a lot of the politics going on here in ways users would neither know or care about, I imagine. Probably just that we added these new APIs and leave it at that.

Did the old `Start*` methods ever GA? I don't recall. If so, they should just be hidden and we typically mention that along with their replacement, which can work here. E.g.,

```
- Added `TextAnalyticsClient.Extractsummary` and `ExtractSummaryAsync`.
- Hid `TextAnalyticsClient.StartExtractSummary` and `StartExtractSummaryAsync` in favor of `ExtractSummary` and `ExtractSummaryAsync`. These methods can still be used but are not shown in code suggestions.
```

Reply:
I will update the phrasing with this suggestion.

For more context, the `Start*` methods didn't GA in the specific case of summarization (ExtractiveSummarize and AbstractiveSummarize), so I removed those entirely. But I did hide the other ones that had previously GA'd, e.g. RecognizeCustomEntities, MultiLabelClassify, etc.